### PR TITLE
Return a default wallet profile absent an env var for it

### PR DESCRIPTION
**Pull request by chilanti**
### Checklist:

- [ ] Read the [Code of Conduct](https://github.com/appsody/website/blob/master/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md).

- [ ] Followed the [commit message guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).

- [ ] Stack adheres to [Appsody stack structure](https://github.com/appsody/website/blob/master/content/docs/stacks/stacks-overview.md#stack-structure).

### Modifying an existing stack:

- [ ] Updated the stack version in `stack.yaml`

### Contributing a new stack:

- Describe how application dependencies are managed:

- Explain how Appsody file watcher is utilized:

- Describe other Appsody environment variables defined by the stack image:

- Describe any limitations and known issues:

### CODE REVIEW

File: incubator/blockchain-client-openliberty/templates/default/src/main/java/application/cm/ConnectionConfiguration.java [hunk 1]
```diff
@@ -22,7 +23,11 @@ public static String getConnectionProfile() {
     }
 
     public static String getWalletProfile() {
-        return System.getenv(FABRIC_WALLET_PROFILE_ENV_VAR);
+        String walletProfile = System.getenv(FABRIC_WALLET_PROFILE_ENV_VAR);
+        if (walletProfile == null || walletProfile == "") {
```
Comment (rykal-nate) [inline incubator/blockchain-client-openliberty/templates/default/src/main/java/application/cm/ConnectionConfiguration.java:27]: Nit - Use walletProfile.isEmpty() here.

**Comment by rykal-nate** [comment]
@chilanti - Is this related to the automated stack testing?

**Review by rykal-nate — APPROVED** [review]
Approved ... please see minor comments.

**Comment by chilanti** [comment]
This is needed because we do not expect the "wallet profile" environment variable to be generated by the VSCode "wizard" that creates the env var file.

**Pull request merged**